src/components/NewOrder.js

```diff
@@ -1,26 +1,76 @@
 import { Component, DOM as dom } from 'react';
 import { connect } from 'react-redux';
 import * as actions from '../action_creators';
+import _ from 'underscore';
```

Comment:
Yo you should check out ramda too. More 'purely' functional, automatically curried functions, easier composition, etc..

http://fr.umio.us/why-ramda/